guides/writing-addons/index.md

```diff
@@ -29,17 +29,28 @@ This directory can hold many of the same subdirectories and files that an Ember
 #### `app/`
 
 The `app` directory plays an important role to help an Ember app automatically discover the components exported by an addon.
-The default way to make a component is to put the implementation in `addon/`, which allows developers to import and extend the addon component. However, Ember apps always look for components within the `app` namespace, so we must re-export our components from `app/`.
+The default way to make a component is to put the implementation in `addon/`, which allows developers to import and extend the addon component. However, Ember apps always look for components within the `app` namespace, so you must re-export our components from `app/`.
 
-Fortunately, when we run `ember generate component my-component-name` in an addon project, the CLI takes care of all this re-exporting business. It creates the necessary files and code for us. Addon authors don't usually need to think about the `app` directory or do any work in it.
+Fortunately, when you run `ember generate component my-component-name` in an addon project, the CLI takes care of all this re-exporting business. It creates the necessary files and code for us. Addon authors don't usually need to think about the `app` directory or do any work in it.
+
+#### `public/`
+
+You can add static assets in the `public/` directory in your addon, and it will automatically be
+merged into the public directory in the host app under `/your-addon-name/`.
+For example, if you add an image at `public/icon.png`, the host app can reference it as
+`/your-addon-name/icon.png` in CSS or JS.
+
+Note that if your addon is published into an npm restricted scope, the final path of the asset also
+includes that. For example, if your addon is called `@foo/my-awesome-addon`, the same image file
+described above will be available at `/@foo/my-awesome-addon/icon.png`.
```

Comment:
I thought about making this a "Zoey says", but didn't see it used anywhere else and didn't want to mess around with making it available. Would be cool to make the infra available in this repo too

Reply:
This will probably happen with the redesign efforts, but it's something to keep in mind!